nocrash.py

```diff
@@ -59,6 +60,19 @@ def error(message):
     logging.error('[NoCrash] {}'.format(message))
 
 
+if 'no-git-user-check' in persistent_arguments:
+    persistent_arguments.remove('no-git-user-check')
+else:
+    git_name = git.config('--get', 'user.name', _ok_code=[0, 1])
+    if git_name != "SmokeDetector":
+        logging.error('git config user.name "{0}" is wrong'.format(git_name))
```

Comment:
Maybe mention that the error can be bypassed; e.g. `git config user.name "{0}" is wrong; use no-git-check to ignore`

Reply:
Yeah, I convinced myself that short source lines would be more important than useful feedback, which is obviously not a good balance.  Hang on.